commons-jcs-jcache/src/main/java/org/apache/commons/jcs/jcache/cdi/CDIJCacheHelper.java

```diff
@@ -393,7 +432,10 @@ private CacheResolverFactoryImpl defaultCacheResolverFactory()
             if (defaultCacheResolverFactory != null) {
                 return defaultCacheResolverFactory;
             }
-            defaultCacheResolverFactory = new CacheResolverFactoryImpl();
+            defaultCacheManager=instance(CacheManager.class);
+            defaultConfigurationResolver=instance(ConfigurationResolver.class);
```

Comment:
JCache configuration is also hardcoded (as it was before) but now it can be replaced by an injected bean.